packages/preview-middleware/src/base/flp.ts

```diff
@@ -411,12 +425,18 @@
                     this.logger
                 );
                 if (success) {
-                    res.status(200).send(message);
+                    res.writeHead(200);
+                    res.write(message);
```

Comment:
@tobiasqueck do we need something like [sanitizer](https://www.npmjs.com/package/sanitizer) to fix this issue or is this a false positive as `message` comes from `writeChange` and is actually `'FILE_CREATED ${fileName}.${fileType'`. Nevertheless I'm not sure what could happen in case of malicious file names 🙈  At least the issue must have been existing the whole time as I didn't change the general setup.